R/addin.R

```diff
@@ -69,29 +72,83 @@ confl_create_post_from_Rmd <- function(Rmd_file = NULL, interactive = NULL,
     env = globalenv()
   )
 
+  # set confl setting
   front_matter <- rmarkdown::yaml_front_matter(Rmd_file, "UTF-8")
 
+  front_matter_confl <- front_matter$confl_setting
```

Comment:
What about `conflr_settings`? `confl_` is a common prefix used internally, but I feel `conflr` fits here.

Reply:
You're right.Correct.